src/traces/cone/attributes.js

```diff
@@ -64,20 +64,37 @@ var attrs = {
         description: 'Sets the z components of the vector field.'
     },
 
-    vx: {
-        valType: 'data_array',
-        editType: 'calc',
-        description: 'Sets the x coordinates of the vector field mesh.'
-    },
-    vy: {
-        valType: 'data_array',
-        editType: 'calc',
-        description: 'Sets the y coordinates of the vector field mesh.'
-    },
-    vz: {
-        valType: 'data_array',
+    cones: {
```

Comment:
@alexcjohnson I went with `cones.(x|y|z)` which looks a lot like attribute containers `contours` and `bins`. Hope you'll like it.